libs/ledger-live-common/src/families/xrp/platformAdapter.ts

```diff
@@ -1,14 +1,14 @@
 // TODO: to remove once live-app-sdk is depreciated and removed from LL
 
 import { RippleTransaction as PlatformTransaction } from "@ledgerhq/live-app-sdk";
-import { Transaction } from "./types";
+import { Transaction } from "@ledgerhq/coin-xrp/types";
 
 const CAN_EDIT_FEES = true;
 
 const areFeesProvided = (tx: PlatformTransaction): boolean => !!tx.fee;
 
 const convertToLiveTransaction = (tx: PlatformTransaction): Partial<Transaction> => {
-  return tx;
+  return { ...tx, family: "xrp" };
```

Comment:
Not sure why we need to add `family` here as I would expect it to already be in the `tx`
I suppose it's for making sure you always have the correct one

Reply:
Ok got it makes sense, thx for the details